frontend/tests/unit/ignore_button.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { mount, shallowMount } from "@vue/test-utils";
+import IgnoreButton from "@/components/IgnoreButton.vue";
+
+const mountIgnoreButtonComponent = async (textButton: string = "ignorer") => {
+  const ignoreButton = shallowMount(IgnoreButton, {
+    props: {
+      textButton: textButton,
+    },
+  });
+
+  return ignoreButton;
+};
+
+describe("IgnoreButton", () => {
+  describe("when component is displayed", () => {
+    it("should be a button", async () => {
```

Comment:
We can remove design tests or check if button is contained within the component.